tooling/src/hypothesistooling/__main__.py

```diff
@@ -271,7 +271,7 @@ def update_python_versions():
     install.ensure_python(PYTHONS[ci_version])
     where = os.path.expanduser("~/.cache/hypothesis-build-runtimes/pyenv/")
     subprocess.run(
-        "git fetch && git reset --hard origin/master",
+        "git fetch && git reset --hard origin/main",
```

Comment:
pyenv again.  It's pretty weird that our scripts operate an entirely separate git repo, but here we are.